src/components/Header.tsx

```diff
@@ -60,10 +60,6 @@ export default function Header() {
                   handleMouseEnter();
                   changeHeaderContent(section.text);
                 }}
-                onMouseLeave={() => {
-                  handleMouseLeave();
-                  changeHeaderContent(section.text);
-                }}
```

Comment:
Why was this removed completely? I thought you want to know whether when mouse leaves button?

Reply:
addressed in different PR.